### PR TITLE
Fix: Set ONNX Options: Disable telemetry, limit threads, lower CPU usage

### PR DESCRIPTION
**Pull request by SnuffSocket**

### CODE REVIEW

**Comment by dfgHiatus** [comment]
Thanks for your PR! Could you please elaborate on what you mean by "potential memory leak"?

**Comment by RamesTheGeneric** [comment]
Heya, what is the reason for hard coding the thread count at 1?

**Comment by dfgHiatus** [comment]
I've added some feedback on your PR. As with Rames's comment, did you see any performance change(s) with a thread count of 1? Thanks again

**Comment by SnuffSocket** [comment]
> Heya, what is the reason for hard coding the thread count at 1?

That's from me testing "ORT_PARALLEL" mode.
Restricting "inter_op_num_threads" didn't seem be the limiting factor, unlike "intra_op_num_threads".
And with default "0" and "ORT_SEQUENTIAL" the runtime decided it should be "1" anyway.

So to avoid any overhead with letting runtime decide what to set it, may as well set it ourselves?

**Comment by SnuffSocket** [comment]
> I've added some feedback on your PR. As with Rames's comment, did you see any performance change(s) with a thread count of 1? Thanks again

I can't see the feedback?

**Comment by dfgHiatus** [comment]
Oh, I have some pending changes I marked for review. You can see them under the "Files changed" tab

**Comment by SnuffSocket** [comment]
Yeah I still can't see 'em, did you forget to submit them? :/

**Review by dfgHiatus — CHANGES_REQUESTED** [review]

**Comment by dfgHiatus** [comment]
Aaand I did. That's two for two, sorry :>

File: BabbleApp/landmark_processor.py [hunk 1]
```diff
@@ -88,10 +88,13 @@ def __init__(
         #self.calibrate_config = np.empty((1, 45))
         #self.min_max_array = np.empty((2, 45))
 
-
+        ort.disable_telemetry_events()
         self.opts = ort.SessionOptions()
+        self.opts.inter_op_num_threads = 1
```
[the remark this answers is not in the document]
Comment (SnuffSocket) [reply BabbleApp/landmark_processor.py:93]: I disagree as the current/default behaviour without it aka "0" sets it to 1, also the default "ORT_SEQUENTIAL" mode doesn't benefit from giving this more threads anyway..

File: BabbleApp/babble_processor.py [hunk 2]
```diff
@@ -87,9 +87,13 @@ def __init__(
         self.calibrate_config = np.empty((1, 45))
         self.min_max_array = np.empty((2, 45))
 
+        ort.disable_telemetry_events()
         self.opts = ort.SessionOptions()
+        self.opts.inter_op_num_threads = 1
         self.opts.intra_op_num_threads = settings.gui_inference_threads
```
[the remark this answers is not in the document]
Comment (SnuffSocket) [reply BabbleApp/babble_processor.py:95]: I disagree as the current/default behaviour without it aka "0" sets it to 1, also the default "ORT_SEQUENTIAL" mode doesn't benefit from giving this more threads anyway..

**Review by dfgHiatus — APPROVED** [review]

**Pull request merged**